src/types.js

```diff
@@ -19,18 +79,46 @@ export const TYPES = {
     name: 'Community Guidelines',
     fileName: 'community_guidelines'
   },
+  customerPrivacyPolicy: {
+    name: 'Costomer Privacy Policy',
+    fileName: 'costumer_privacy_policy'
+  },
+  DMCA: {
+    name: 'DMCA',
+    fileName: 'dmca'
+  },
+  EULA: {
+    name: 'EULA',
+    fileName: 'eula'
+  },
   brandGuidelines: {
     name: 'Brand Guidelines',
     fileName: 'brand_guidelines'
   },
+  geoGuidelines: {
+    name: 'Geo Guidelines',
+    fileName: 'geo_guidelines'
+  },
   reviewGuidelines: {
     name: 'Review Guidelines',
     fileName: 'review_guidelines'
   },
+  cookiePolicy: {
+    name: 'Cookie Policy',
+    fileName: 'cookie_policy'
+  },
+  cookieAndAdPolicy: {
```

Comment:
In favor of merging with `cookiesPolicy`, unless I misunderstand and there is a difference.